File: Assets/Script/MatchmakingManager.js
#pragma strict
import UnityEngine.Networking;
import UnityEngine.Networking.Match;
 
 
 
public class MatchmakingManager extends NetworkManager{
 
    private var networkMatch:NetworkMatch;
 
     function Update(){
         if(networkMatch == null){
             var nm = GetComponent(NetworkMatch);
             if(nm != null){
                 networkMatch = nm as NetworkMatch;
                 var appid:UnityEngine.Networking.Types.AppID;
                 appid= 94451;
                 networkMatch.SetProgramAppID(appid);
             }
         }
     }
}

File: Assets/Script/MatchmakingManager.cs
using UnityEngine;
using System.Collections;
using UnityEngine.Networking;
using UnityEngine.Networking.Match;
using UnityEngine.Networking.Types;

public class MatchmakingManager : NetworkManager {

    private NetworkMatch networkMatch;

    void Update()
    {
        if (networkMatch == null)
        {
            NetworkMatch nm = GetComponent<NetworkMatch>();
            if (nm != null)
            {
                networkMatch = nm as NetworkMatch;
                AppID appid;
                //appid = 94451;
                //networkMatch.SetProgramAppID(appid);
                
            }
        }
    }
}
